src/mixins/filing-template-mixin.ts

```diff
@@ -171,7 +191,19 @@ export default class FilingTemplateMixin extends Vue {
     this.setNameRequest(filing.incorporationApplication.nameRequest)
 
     // Set Name Translations
-    this.setNameTranslations(filing.incorporationApplication.nameTranslations?.new)
+    if (filing.incorporationApplication.nameTranslations instanceof Array) {
```

Comment:
If I understand correctly, this is to handle a couple of possible structures here. Please add a comment explaining what you expect to see here (your assumption).

Is there any way to NOT have this situation? Is it specifically to handle a real IA vs a draft? If feels complicated.